src/scenes/home/home.js

```diff
@@ -56,7 +57,7 @@ class Home extends Component {
   }
 
   setBgImage(location) {
-    if (location.pathname === '/') {
+    if (location.pathname === '/' || location.pathname === '/families') {
```

Comment:
This can be removed if we don't want the `BgImage` on the families page.

Reply:
We can leave it in here. I'll be fiddling with the Nav this weekend and this page isn't linked anywhere on the site.

While I'm in the Nav, I'll add a different image and try to set it up to dynamically choose a header image based upon route.